WeScan/Review/ReviewViewController.swift

```diff
@@ -71,10 +108,24 @@ final class ReviewViewController: UIViewController {
     
     // MARK: - Actions
     
+    @objc private func toggleEnhancedImage() {
+        guard enhancedImageIsAvailable else { return }
+        if isCurrentlyDisplayingEnhancedImage {
+            imageView.image = results.scannedImage
+            enhanceButton.tintColor = .white
+        } else {
+            imageView.image = results.enhancedImage
+            enhanceButton.tintColor = UIColor(red: 64 / 255, green: 159 / 255, blue: 255 / 255, alpha: 1.0)
```

Comment:
We should use the navigationbar tint color here, to make it customizable.

Reply:
I don't think this is a good idea? If the tint color is, for example, dark blue, it'll work on the navigation bar but not on the (dark) toolbar.

I do think we can add an actual 'tint color' when we add preferences though.

Reply:
Yeah that's true. Until we have those preferences, we don't have a good way of customizing things. This is something pretty important to pick up in the future.

Reply:
👍